docs/operations/comanage-recipes.md

```diff
@@ -0,0 +1,71 @@
+COManage Recipes
+==============================
+
+A collection of step-by-step instructions for common actions for use by administrators of the OSG COManage.
+
+
+Steps for Common COManage Actions
+----
+This section contains some common actions administrators perform on the OSG COManage, and how to do so.
+
+### Provisioning a CO Group in COManage
+In order for a CO Group from COManage to show up in LDAP (and thus be made available for reference on Hosts),
+ it must first be provisioned.
```

Comment:
I think it'd be worth adding an admonition here to speak to the fact that OIDC clients needing groups don't need any of these steps outside of the group creation (see https://osg-htc.org/technology/documentation/style-guide/#notes and https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types)

```
```